pages/nation.md

```diff
@@ -6,7 +6,7 @@
 
 ## Introduction
 
-In step 4, you registered your community BeLL with the nation. Now, you will learn how to keep your community BeLL in sync with the nation.
+In [step 4](#!./pages/bellapps.md), you registered your community BeLL with the nation. Now, you will learn how to keep your community BeLL in sync with the nation.
```

Comment:
I think community registration is actually in [Step 1 - BeLL Installation - Configurations](http://open-learning-exchange.github.io/#!./pages/configurations.md).